dockerfile/hooks/build

```diff
@@ -15,10 +15,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
 echo "------ dolphinscheduler start - build -------"
 printenv
 
-docker build --build-arg version=$version --build-arg tar_version=$tar_version  -t $DOCKER_REPO:$version .
+echo -e "Current Directory is $(pwd)\n"
+
+# maven package(Project Directory)
+echo -e "mvn clean compile package -Prelease"
+mvn clean compile package -Prelease
+
+# mv dolphinscheduler-bin.tar.gz file to dockerfile directory
+echo -e "mv $(pwd)/dolphinscheduler-dist/target/apache-dolphinscheduler-incubating-${VERSION}-SNAPSHOT-dolphinscheduler-bin.tar.gz $(pwd)/dockerfile/\n"
```

Comment:
It is better to build image based on the current branch code, both CI and release can be used, rather than build based on the package

Reply:
`User` can use it.